src/main/java/com/google/javascript/clutz/PlatformSymbols.java

```diff
@@ -0,0 +1,1716 @@
+package com.google.javascript.clutz;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * There are many "platform" symbols, where Closure defines them in an externs file and TypeScript
+ * defines them in a lib.d.ts file. When a Closure file references a platform type like HTMLElement
+ * or platform global like encodeURIComponent, rather than referring to a type that Clutz derives
+ * from the externs we'd rather it refer directly to the TypeScript type.
+ *
+ * <p>Note that doing this is not strictly correct (the two may have slightly different definitions)
+ * but this is likely what users expect and it reduces the output size.
+ *
+ * <p>This class maintains all the metadata about these types; for example, Closure and TypeScript
+ * have different names for the interface that is the 'style' property on an element, so this class
+ * has a map from Closure name to TypeScript name.
```

Comment:
add general guidance on adding and removing symbols in this class

Reply:
Can you elaborate?  I think the local comments are sufficient maybe?

Reply:
Imagine x months in the future: lib.d.ts and platform externs have changes. How does one go about updating this lists.

Can we write a simple flow script

- if Symbol X appears in lib.d.ts and
  - it is not in closure and someone needs to use it google add it to ...
  - it is in closure, add it to ...
  - it is not is closure and noone needs to use it do nothing.

- if symbol x appears in closure externs
 - ...

Maybe I am overthinking it ...